tools/submission/submission-checker.py

```diff
@@ -0,0 +1,554 @@
+
+
+MODEL_CONFIG = {
+    "v0.5": {
+        "models": ["ssd-small", "ssd-large", "mobilenet", "resnet", "gnmt"],
+        "required-scenarios-datacenter": {
+            # anything goes
+        },
+        "required-scenarios-edge": {
+            # anything goes
+        },
+        "accuracy-target": {
+            "mobilenet": ("acc", 71.68 * 0.98),
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-small": ("mAP", 22 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "gnmt": ("bleu", 23.9 * 0.99),
+        },
+        "performance-sample-count": {
+            "mobilenet": 1024,
+            "resnet": 1024,
+            "ssd-small": 256,
+            "ssd-large": 64,
+            "gnmt": 3903900,
+        },
+        "seeds": {
+            "qsl_rng_seed": 3133965575612453542,
+            "sample_index_rng_seed": 665484352860916858,
+            "schedule_rng_seed": 3622009729038561421,
+        },
+    },
+    "v0.7": {
+        "models": ["ssd-large", "resnet", "rnnt", "3d-unet", "dlrm", "bert"],
+        "required-scenarios-datacenter": {
+            "resnet": ["Server", "Offline"],
+            "ssd-large": ["Server", "Offline"],
+            "rnnt": ["Server", "Offline"],
+            "bert": ["Server", "Offline"],
+            "dlrm": ["Server", "Offline"],
+            "3d-unet": ["Offline"],
+        },
+        "required-scenarios-edge": {
+            "resnet": ["Server", "Offline"],
+            "ssd-large": ["Server", "Offline"],
+            "rnnt": ["Server", "Offline"],
+            "bert": ["Server", "Offline"],
+            "dlrm": ["Server", "Offline"],
+            "3d-unet": ["Offline"],
+        },
+        "accuracy-target": {
+            "resnet": ("acc", 76.46 * 0.99),
+            "ssd-large": ("mAP", 20 * 0.99),
+            "rnnt": ("WER", 7.452 * 0.99),
+            "bert": ("F1", [90.874 * 0.99, 90.874 * 0.999]),
+            "dlrm": ("AUC", [76.46 * 0.99, 76.46 * 0.999]),
+            "3d-unet": ("mean", [0.853 * 0.99, 0.853 * 0.999]),
+        },
+        "performance-sample-count": {
+            "ssd-large": 64,
+            "resnet": 1024,
+            "rnnt": 2513,
+            "bert": 3903900,
+            "dlrm": 204800,
+            "3d-unet": 16,
+        },
+        "seeds": {
+            "qsl_rng_seed": 3133965575612453542,
+            "sample_index_rng_seed": 665484352860916858,
+            "schedule_rng_seed": 3622009729038561421,
+        },
+    },
+}
+
+VALID_DIVISIONS = ["open", "closed"]
+REQUIRED_PERF_FILES = ["mlperf_log_accuracy.json", "mlperf_log_summary.txt", "mlperf_log_detail.txt"]
+REQUIRED_ACC_FILES = REQUIRED_PERF_FILES + ["accuracy.txt"]
+REQUIRED_MEASURE_FILES = ["mlperf.conf", "user.conf", "README.md"]
+TO_MS = 1000 * 1000
+
+MODEL_MAPPING = {
+    "ssd-mobilenet": "ssd-small",
+    "ssd-resnet34": "ssd-large",
+    "resnet50": "resnet"
+}
+
+RESULT_FIELD = {
+    "Offline": "Samples per second",
+    "Single": "90th percentile latency (ns)",
```

Comment:
Should these be changed to `SingleStream` and `MultiStream` to match official names for the scenarios?